src/request/src/cache.js

```diff
@@ -70,9 +68,9 @@ export default class CacheRequest extends Request {
 
   set url(urlString) {
     super.url = urlString;
-    let pathname = global.decodeURIComponent(url.parse(urlString).pathname);
-    const urlParts = pathname.replace(/^\//g, '').split('/');
-    // "pathname" has the following form "/namespace/appKey/collection/id"
+    const pathname = global.decodeURIComponent(url.parse(urlString).pathname);
+    const urlParts = pathname.replace(/^\//, '').split('/');
```

Comment:
@georgiwe Ok makes sense. I have to admit I am not the best at regular expressions but I did use a tester to see the difference and I agree the `g` flag is not needed.